docs/components/data_tables.md

```diff
@@ -0,0 +1,407 @@
+* [Demo site](https://gallery.flutter.dev/#/demo/data-table)
+
+![Demo table without Theme](assets/data_tables/data_table_no_theme.png)
+
+```dart
+
+
+class DataTableDemo extends StatefulWidget {
+  @override
+  _DataTableDemoState createState() => _DataTableDemoState();
+}
+
+class _DataTableDemoState extends State<DataTableDemo> {
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Data Tables'),
+      ),
+      body: ListView(
+        padding: const EdgeInsets.all(16),
+        children: [
+          PaginatedDataTable(
+            header: Text('Header Text'),
+            rowsPerPage: 4,
+            columns: [
+              DataColumn(label: Text('Header A')),
+              DataColumn(label: Text('Header B')),
+              DataColumn(label: Text('Header C')),
+              DataColumn(label: Text('Header D')),
+            ],
+            source: _DataSource(context),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Row {
+  _Row(
+    this.headerA,
+    this.headerB,
+    this.headerC,
+    this.headerD,
+  );
+
+  final String headerA;
+  final String headerB;
+  final String headerC;
+  final int headerD;
+
+  bool selected = false;
+}
+
+class _DataSource extends DataTableSource {
+  _DataSource(this.context) {
+    _rows = <_Row>[
+      _Row('Cell A1', 'CellB1', 'CellC1', 1),
+      _Row('Cell A2', 'CellB2', 'CellC2', 2),
+      _Row('Cell A3', 'CellB3', 'CellC3', 3),
+      _Row('Cell A4', 'CellB4', 'CellC4', 4),
+    ];
+  }
+
+  final BuildContext context;
+  List<_Row> _rows;
+
+  int _selectedCount = 0;
+
+  @override
+  DataRow getRow(int index) {
+    assert(index >= 0);
+    if (index >= _rows.length) return null;
+    final row = _rows[index];
+    return DataRow.byIndex(
+      index: index,
+      selected: row.selected,
+      onSelectChanged: (value) {
+        if (row.selected != value) {
+          _selectedCount += value ? 1 : -1;
+          assert(_selectedCount >= 0);
+          row.selected = value;
+          notifyListeners();
+        }
+      },
+      cells: [
+        DataCell(Text(row.headerA)),
+        DataCell(Text(row.headerB)),
+        DataCell(Text(row.headerC)),
+        DataCell(Text(row.headerD.toString())),
+      ],
+    );
+  }
+
+  @override
+  int get rowCount => _rows.length;
+
+  @override
+  bool get isRowCountApproximate => false;
+
+  @override
+  int get selectedRowCount => _selectedCount;
+}
+
+```
+
+### Key properties
+
+The following is an anatomy diagram of a data table:
+
+![Data table anatomy diagram](assets/data_tables/data-table-anatomy.png)
+
+A data table consists of the following properties:
+
+1. Container
+1. Column header names
+1. Sort button
+1. Row checkboxes
+1. Rows
+
+### Container
+
+| &nbsp; | Property |
+| --- | --- |
+| **Color** | style |
+| **Stroke color** | while setting `ThemeData` fill out `divideColor` parameter |
+| **Stroke width** | N/A |
```

Comment:
Remove N/A properties as per the last few PRs we've done

Reply:
Done